server/src/main/java/org/apache/cassandra/sidecar/acl/AuthCache.java

```diff
@@ -133,7 +133,7 @@ private LoadingCache<K, V> initCache()
     private void configureSidecarServerEventListener()
     {
         EventBus eventBus = vertx.eventBus();
-        eventBus.localConsumer(ON_ALL_CASSANDRA_CQL_READY.address(), message -> warmUpAsync(config.warmupRetries()));
+        eventBus.localConsumer(ON_SIDECAR_SCHEMA_INITIALIZED.address(), message -> warmUpAsync(config.warmupRetries()));
```

Comment:
Because we were not starting warm up on `ON_SIDECAR_SCHEMA_INITIALIZED`, first few tries at warming up at failing. Hence switched to starting on schema initialized